api_v3/services/UserEntryService.php

```diff
@@ -94,10 +95,6 @@ public function deleteAction($id)
 	 */
 	public function listAction(KalturaUserEntryFilter $filter, KalturaFilterPager $pager = null)
```

Comment:
Question - can there be a case that filter not supplied to the listAction? if not why allow it